apps/sertra/components/mod-selector/search.tsx

```diff
@@ -105,7 +103,7 @@ export const SearchBox = <T,>({
       </div>
       {isFocused && (
         <div className={styles.optionsAnchor}>
-          <div className={styles.optionsBox}>
+          <div className={styles.optionsBox} ref={optionsRef}>
```

Comment:
I guess this approach is fine, but I'm wondering if we could bind the ref directly to the element were interested in with something like this:

```
{options.map((option, i) => (
  <button
    ref={i === 0 ? optionRef : null}
    className={styles.optionButton}
    key={keyExtractor(option)}
    onKeyDown={(e) => listKeyDownHandler(option, e)}
    onClick={(e) => onClickHandler(option, e.currentTarget)}
  >
    {renderOption(option)}
  </button>
)}
```

Haven't tested, might not work 🤷🏻